stories/components/FsForm.vue

```diff
@@ -48,12 +54,18 @@ FForm(
         name="FTextarea"
         hint="Hint for FTextarea"
       )
-    FGridItem(span="12")
+    FGridItem.FForm__actions(span="12")
```

Comment:
Ton composant étant un composant pour storybook et nommé FsForm, alors la classe :
```suggestion
    FGridItem.FsForm__actions(span="12")
```